working/v3.0.0-draft2/client-authentication/jwt-bearer-auth-schema.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://apicentre.paymentsnz.co.nz/schemas/jwt-bearer-auth-schema.json",
+  "type": "object",
+  "title": "The schema used with JWT authentication",
+  "description": "The schema to be used with private_key_jwt as specified by OIDC core final 1.0",
```

Comment:
Should this be FAPI?

Reply:
It is originally specified in [OIDC Core, section 9](https://openid.net/specs/openid-connect-core-1_0.html#ClientAuthentication) as a client authentication option.  [FAPI restricts the options](https://openid.net/specs/openid-financial-api-part-2-1_0.html#authorization-server) to `private_key_jwt` and other authentication methods, the NZ Security Profile then further restricts to `private_key_jwt`.